<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalAndRemoteDifferentWidget</class>
 <widget class="QWidget" name="LocalAndRemoteDifferentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QLabel" name="selectLabel">
     <property name="font">
      <font>
       <family>Lato 13</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#selectLabel
{
    font-family: Lato Bold;
    font-size: 13px;
    line-height: 16px;
    color: #333333;
}</string>
     </property>
     <property name="text">
      <string>Please select which version you want to keep</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="chooseContainer" native="true">
     <layout class="QVBoxLayout" name="chooseLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="CloudStalledIssueChooseWidget" name="chooseRemoteCopy" native="true"/>
      </item>
      <item>
       <widget class="LocalStalledIssueChooseWidget" name="chooseLocalCopy" native="true"/>
      </item>
      <item>
       <widget class="GenericChooseWidget" name="keepLastModifiedOption" native="true"/>
      </item>
      <item>
       <widget class="GenericChooseWidget" name="keepBothOption" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CloudStalledIssueChooseWidget</class>
   <extends>QWidget</extends>
   <header>StalledIssueChooseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LocalStalledIssueChooseWidget</class>
   <extends>QWidget</extends>
   <header>StalledIssueChooseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GenericChooseWidget</class>
   <extends>QWidget</extends>
   <header>StalledIssueChooseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
